File: Week_2/Day_4/Défis_du_Jour/Defis_du_jour.js
//Demande de mots à l'utilisateur
let chaine = prompt("Svp entrer plusieurs mots et séparez-les par des virgules");
//Rangements de ces mots dans un tableau
chaine = chaine.split(",");
//Affichage des mots dans un cadre rectangulaire
let affich=0;
for(let mot of chaine){
    if(affich<mot.length)
    {
        affich=mot.length;
    }
}

for(let i=0;i<chaine.length;i++)
{
    
    if(i==0) 
    {
        console.log("*".padStart((max+4),"*"));
    }
        console.log("* " +chaine[i].concat("".padStart(((max+4)-2)-(2+chaine[i].length)), " ") +"*");
    if(i==chaine.length-1)
    {
        console.log("*".padStart((max+4),"*"));
    }
}

File: Week_4/Day_5/Defis du jour/Defis_du_jour.py
Chaine = input("Svp, entrer un ensemble de mots séparé par des virgules. Exemple:||without,hello,bag,world||==>")
Chaine = Chaine.split(',')
Chaine = sorted(Chaine)
New_Chaine = ""
for mot in Chaine:
    New_Chaine += mot + ','
print(New_Chaine)
